clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/ClioWebClient.scala

```diff
@@ -91,16 +91,19 @@ class ClioWebClient(
   }
 
   def queryWgsUbam(
-    input: TransferWgsUbamV1QueryInput
+    input: TransferWgsUbamV1QueryInput,
+    includeDeleted: Boolean = false
   )(implicit bearerToken: OAuth2BearerToken): Future[HttpResponse] = {
     val entity =
       HttpEntity(
         ContentTypes.`application/json`,
         input.asJson.pretty(implicitly[Printer])
       )
+    val queryPath = if (includeDeleted) "queryall" else "query"
```

Comment:
I don't think it'd be any less work to use a query parameter instead. We'd get rid of this `if`-`else` but then gain a new one in the server to check whether or not to modify the query input.

Reply:
This would let us delete an entire endpoint in the server, so the server would now have less code.

If we changed this to send `DocumentStatus` instead of a boolean then we wouldn't need to do any work on the server side either.

Reply:
Good point, I didn't think about the fact that we'd get to delete the queryall directive and its associated boilerplate.

I think we still want the query vs queryall logic to happen in the server (vs. the client) so query results are the same regardless of the client used / without needing to duplicate the logic across all clients (which is a real worry since ops would rather use an auto-generated Python client).

In any case, I don't think changing the server API is in scope for this PR.

Reply:
Clients already need logic to determine which endpoint to hit. How is this different than logic that would provide a different argument to an endpoint?

Regardless of what's done for python, changing this to use `DocumentStatus` seems like a clear win to me, since we have to provide logic anyway.